Tests/SentryTests/Integrations/Performance/AppStartTracking/SentryAppStartTrackerTests.swift

```diff
@@ -271,15 +286,15 @@ class SentryAppStartTrackerTests: NotificationCenterTestCase {
         assertValidHybridStart(type: .warm)
     }
     
-    private func givenPreviousAppState(appState: SentryAppState) {
```

Comment:
As this is not the previous but the current app state, this name really didn't make sense.